src/Cms/System.php

```diff
@@ -514,6 +514,18 @@ public function sessions(): bool
         return is_writable($this->app->root('sessions'));
     }
 
+    /**
+     * Returns the site's title as defined in the
+     * content file or `site.yml` blueprint
+     *
+     * @return string
+     */
+    public function title(): string
+    {
+        $blueprint = $this->app->site()->blueprint()->title();
+        return $this->app->site()->title()->or($blueprint)->value();
```

Comment:
It's probably never going to be performance-relevant, but I would still improve it a bit. 

```suggestion
        $site = $this->app->site();

        if ($site->title()->isNotEmpty()) {
          return $site->title()->value();
        }
        
        return $site->blueprint()->title();
```